extensions/arc/deployment/src/main/resources/dev-ui/arc/qwc-arc-beans-dependency.js

```diff
@@ -0,0 +1,98 @@
+import { LitElement, html, css} from 'lit';
+import { arcState } from './arc-state.js';
+import { dependencyGraphs } from 'arc-data';
+import 'qui-d3-force';
+
+/**
+ * This component shows the Arc Beans Dependency
+ */
+export class QwcArcBeansDependency extends LitElement {
```

Comment:
`QwcArcBeansDependency` -> `QwcArcBeanDependencyGraph` or at least `QwcArcBeanDependencies`. And the same applies to the file name.